nuxt.config.js

```diff
@@ -1,7 +1,7 @@
 export default {
   // Global page headers: https://go.nuxtjs.dev/config-head
   head: {
-    title: 'goals-site',
+    title: 'Goals',
```

Comment:
Can we please put a title like `Goals | Real Dev Squad`? Because we are following this pattern for members and status sites too. So keeping the pattern same will be good.

Reply:
I have done the changes. so closes #27